uobtheatre/addresses/migrations/0001_initial.py

```diff
@@ -4,7 +4,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
What linting rules are being applied now?

Reply:
The same pylint ones. I don't think these files have ever been touched because you force committed...